src/main/resources/Mobs.json

```diff
@@ -1,11 +1,11 @@
 [
   {
-    "name": "Goblin",
+    "name": "goblin",
```

Comment:
I believe mob names will be capitalized in the file, to match naming in-game.

Reply:
Changed